local-development.md

```diff
@@ -0,0 +1,63 @@
+# Local Deployment Tesing
+
+At times it may be necessary to change the way the deployment works, the most convienent way of doing this is to make changes locally. This avoids having to setup and wait for external services. This file contains some details on how to do this development locally.
+
+# Requirements
+
+You will need installed `Vagrant` and `VirtualBox`
+
+# Getting Started
+
+Clone the repoistory [builder](https://github.com/elifesciences/builder) onto your local machine:
+```
+  git clone git@github.com:elifesciences/builder.git
+```
+
+In the root of the project folder run `vagrant up` and select elife-xpub. Alternatively you can supply this in the `PROJECT` environment variable:
```

Comment:
A link to https://github.com/elifesciences/builder/#vagrant will be authoritative on this, and get updated.